metricbeat/metricbeat.reference.yml

```diff
@@ -377,16 +377,15 @@ metricbeat.modules:
   #raw: false
 
 #-------------------------------- Nginx Module -------------------------------
-#- module: nginx
-  #metricsets: ["stubstatus"]
-  #enabled: true
-  #period: 10s
+- module: nginx
+  metricsets: ["stubstatus"]
+  period: 10s
 
   # Nginx hosts
-  #hosts: ["http://127.0.0.1"]
+  hosts: ["http://127.0.0.1"]
 
   # Path to server status. Default server-status
-  #server_status_path: "server-status"
+  server_status_path: "server-status"
 
```

Comment:
I've looked at the configuration, I see something we define `enabled: false` and sometime we don't, I think we should be explicit and it should be defined all the time. 

I am saying this because your changes is removing that option.

Reply:
Agree, see https://github.com/elastic/beats/issues/6672 We will be cleaning this up going through https://github.com/elastic/beats/issues/6668